data/json/items/gun/9mm.json

```diff
@@ -179,7 +179,7 @@
     "valid_mod_locations": [ [ "accessories", 4 ], [ "barrel", 1 ], [ "bore", 1 ], [ "grip", 1 ], [ "mechanism", 4 ], [ "muzzle", 1 ], [ "rail", 1 ], [ "sights", 1 ] ],
     "magazine_well": 1,
     "magazines": [
-      [ "9mm", [ "glockmag", "glockbigmag" ] ],
+      [ "9mm", [ "glockmag", "glockbigmag" , "glock17_17", "glock17_22", "glock17_33", "glock_drum_50rd", "glock_drum_100rd" ] ], 
```

Comment:
You removed the 33 round mag so it should be taken out here.

Reply:
Whoops!